lab-examples/ixiac01/ixiac01.clab.yml

```diff
@@ -6,8 +6,8 @@ topology:
       kind: keysight_ixia-c-one
       image: ghcr.io/open-traffic-generator/ixia-c-one:0.0.1-2755
```

Comment:
@mfzhsn can you check if the recent ixia-c version would work? This version is quite outdated

ghcr.io/open-traffic-generator/ixia-c-one:0.1.0-84